src/main/resources/assets/immersivepetroleum/manual/en_us/hydrotreater.txt

```diff
@@ -1,13 +1,19 @@
-Sulfur Recovery Unit
+High-Pressure Refinery Unit
 Smell of Fireworks
```

Comment:
Since the name changed, the subtitle could need a snazzy change aswell.

Reply:
Oh, good point!